service.subtitles.legendastv/resources/lib/LTVutilities.py

```diff
@@ -0,0 +1,257 @@
+# -*- coding: utf-8 -*-
+
+import os
+import re
+import sys
+import xbmc
+import xbmcvfs
+import shutil
+import unicodedata
+import urllib, urllib2
+import ssl
+
+try: import simplejson as json
+except: import json
+
+try:
+    __scriptname__ = sys.modules[ "__main__" ].__scriptname__
+except:
+    __scriptname__ = ""
+
+TheTVDBApi = "A1738606AC58C23D"
```

Comment:
I doubt this won't last long. tvdb is planing to drop free accounts

Reply:
I'll try to change this in a future update. The intent of this particular update is to make it work fully on Leia, but I'll add this to my to-do list

Reply:
Sure